src/commands/serve.rs

```diff
@@ -33,10 +33,20 @@ impl Serve {
             },
         );
 
-        let app = Router::new().fallback(static_files_service);
+        let app;
+        let route;
+        if let Some(path_prefix) = config.path_prefix {
+            route = format!("/{}", path_prefix);
```

Comment:
is the slash necessary? my instinct is it wouldn't be but i'm not sure

Reply:
i think this is a scenario where we may want to use `Url` https://docs.rs/url/latest/url/struct.Url.html#method.join

Reply:
oh no

```
thread 'main' panicked at 'Paths must start with a `/`', src/commands/serve.rs:41:18
```

Reply:
well that settles that ha